curated_tokenizers/_spp.pyx

```diff
@@ -34,6 +42,8 @@ cdef class SentencePieceProcessor:
                 protocol buffer.
         """
         cdef SentencePieceProcessor processor = SentencePieceProcessor.__new__(SentencePieceProcessor)
+        if len(protobuf) == 0:
```

Comment:
Maybe add a comment that sentencepiece returns an empty protobuf for an uninitialized model.